src/App.tsx

```diff
@@ -18,50 +18,71 @@ import './App.css'
 
 let checkCardInterval = 0
 
-// I want to use a hook for this, but it's not reacting quickly enough
+// I want to use a hook for these, but it's not reacting quickly enough
 // in the loop to prevent multiple loads of the long value.
 let loadingElection = false
+let electionLoaded = false
```

Comment:
Frustrated, as I think I know how to fix this, but I don't have a dev env which supports smart cards.